tensorflow/contrib/periodic_intersperse/python/kernel_tests/periodic_intersperse_op_test.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import tensorflow as tf
+
+
+class PeriodicIntersperseTest(tf.test.TestCase):
+    def testPeriodicIntersperse(self):
+        from tensorflow.contrib.periodic_intersperse import periodic_intersperse as pi
```

Comment:
minor, could you import this after `import tensorflow as tf` to make it clearer what the dependencies of this test are?